source/_posts/2018-11-13-whining-vs-venting.markdown

```diff
@@ -0,0 +1,20 @@
+
+I don’t like whining. I have two kids who are approaching teenage-dom and I have enough of it in my life already, thank you very much. Whining from adults is especially grating - they should know better, but it still happens.
+
+I view whining as subtly but distinctly different than venting. And what is that difference you ask? Whining focuses on “woe is me!” and doesn’t lend itself to finding solutions. A whiner wants everything and everyone around them to change to accommodate themselves and their needs. Further, whining tends to beget more whining, whereas when a good old fashion vent is complete, progress can be made. Venting can be healthy.
+
+I actually kinda like hearing a good [vent](http://randsinrepose.com/archives/the-update-the-vent-and-the-disaster/) session. If someone is venting to me directly, then it means they have some level of [trust](http://blog.swilliams.me/words/2018/11/01/building-trust-on-teams/) with me. They’re opening up and spilling out a little bit of themselves. This means you need to be wearing your listening hat. Vents can get emotional - this is good, it means the venter is invested deeply in something and that they care. So, listen! With empathy!
+
+As you’re listening (with empathy) consider the context. They have their point of view and you have yours. They are much closer to the ground floor of the situation they are experiencing, but as a manager you Know Things that they may not. You can use this knowledge to nudge the conversation and keep it a vent rather than a whine, but don’t try to fix the problem. Something incredible can happen when someone is venting: by talking through it they may discover a solution all on their own. This is good. If it sounds like therapy, it kind of is! 
+
+Can you turn a whine into a vent? I think you can, but it may be a struggle. Sometimes it may be asking a pointed question: “Oh wow, that *is* a mess… what can we do to tweak it a bit?” Dig in and ask constructive questions, though try to steer things towards the realm of productivity. If you have a strong enough relationship consider plainly stating it, “this almost sounds like a whine, is there a way to frame it so that there is the potential for a solution here?” It may not work, but that’s a whole other ball of wax.
+
+Fortunately, once Ron got it all off his chest he calmed down a bit and evaluated his own communication with the sales team. It wasn’t actually the end of the world (this time) and he picked up a few pointers for how to work a little more effectively with them in the next go around.
```

Comment:
`off his chest he` -> `off his chest, he`

`this time) and he` -> `this time), and he`